lib/routes.json

```diff
@@ -394,6 +394,97 @@
                 "invalidmsg": "",
                 "description": "OAuth token"
             }
+        },
+        "acceptTree": {
+            "application/vnd.github.mockingbird-preview": [
+                "/repos/:owner/:repo/issues/:issue_number/timeline"
```

Comment:
Whoops. I didn't notice these have :owner. I've been meaning to change it, but the library uses :user instead of :owner.